src/ch/interlis/iox_j/filter/Rounder.java

```diff
@@ -106,7 +107,7 @@ private void roundAttrValue(IomObject iomObj, AttributeDef srcAttr) {
                 String attrValue=iomObj.getattrprim(srcAttrName,attri);
                 try {
                     BigDecimal value=roundNumber(attrValue,(NumericType)type);
-                    iomObj.setattrvalue(srcAttrName, value.toString());
+                    ((Iom_jObject)iomObj).setattrvalue(srcAttrName, attri, value.toString());
                 }catch(NumberFormatException ex) {
```

Comment:
Q: @claeis Soll diese Funktion auch ins API aufgenommen werden oder gibt es einen anderen eleganten Weg die Werte mit gleicher Order zu runden und zu schreiben.

Reply:
Man müsste mal allgemein chli nach casts auf Iom_jObject suchen, um herauszufinden, was alles schon gefehlt hat.

Reply:
@claeis Wir haben das Thema mal intern besprochen und sind aktuell der Meinung, dass wir die aktuellen Anpassungen mit Casts realisieren. Die Analyse und Anpassung von iox-api möchten wir in die 2. Phase vom INTERLIS 2.4 Aufwasch verschieben.
CC(@olivergrimm, @patrickackermann)